test/common/testlib.py

```diff
@@ -1339,6 +1340,9 @@ def new_browser(self, machine=None, coverage=False):
             else:
                 if machine.image == reference_image:
                     pixels_label = self.label()
+        # HACK: until @expectedFailurePybridge disappears from all pixel tests
+        if os.environ.get("TEST_SCENARIO") == "pybridge":
```

Comment:
This is slightly incorrect.  It might be the case that we prepare the image with the bridge, but then run tests manually, without setting the scenario...

It's probably "good enough", though.  In fact, it's probably exactly the right sort of pragmatic solution.  After all, it's a hack.

Reply:
Agreed it's a hack. All of that is slated to go away entirely, so I'm not overly worried about it right now. Alternatively the test could also do something like`"python" in m.execute("file /usr/bin/cockpit-bridge")`, but that's quite expensive. Better ideas?

Reply:
I mean it — I think this is fine.  No need to change it.